compliance/sparql/src/test/java/org/eclipse/rdf4j/sail/lmdb/LmdbComplexSPARQLQueryTest.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
This copyright header needs to be changed - even if you copied it from an existing file, it's essentially a new contribution I'd say.